assets/src/animation/parts/pulse/index.js

```diff
@@ -41,8 +41,8 @@ const defaults = {
 
 export function AnimationPulse({
   iterations = 1,
-  scale = 0.5,
-  easing = 'cubic-bezier(0.4, 0.0, 0.2, 1)',
+  scale = 0.05,
```

Comment:
Is this correct? This feels so small, it's hard to see this on small shapes.

Reply:
yea I think it feels small too, but this is the latest for the default setting on the amp-story-animation presets:
https://github.com/ampproject/amphtml/blob/7601993eff6c4399745ae0f6dd46653c7ceaa84e/extensions/amp-story/1.0/animation-presets.js#L117